src/routes/safe/store/actions/transactions/fetchTransactions/index.ts

```diff
@@ -7,10 +7,13 @@ import { loadOutgoingTransactions } from './loadOutgoingTransactions'
 
```

Comment:
please, create a new folder `src/routes/safe/store/actions/transactionsNew`

Reply:
do not merge old and new files

Reply:
fixed